plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeMetadata.java

```diff
@@ -1320,7 +1320,7 @@ public void addColumn(ConnectorSession session, ConnectorTableHandle tableHandle
         DeltaLakeTableHandle handle = checkValidTableHandle(tableHandle);
         checkSupportedWriterVersion(session, handle);
         ColumnMappingMode columnMappingMode = getColumnMappingMode(handle.getMetadataEntry());
-        if (changeDataFeedEnabled(handle.getMetadataEntry()) && CHANGE_DATA_FEED_COLUMN_NAMES.contains(newColumnMetadata.getName())) {
```

Comment:
- this is part of `fixup! Add support for renaming column in Delta Lake` but this changes ADD COLUMN flow; move to separate commit
- update message accordingly. "Column name %s is forbidden when change data feed is enabled" is no longer applicable, since no other name is allowed